### PR TITLE
DOC-3329: Update Corda Network related links

### PR DESCRIPTION
**Pull request by AMcFarlandCampbellR3**

### CODE REVIEW

File: content/en/platform/corda/4.5/open-source/corda-network/corda-network-foundation.md [hunk 1]
```diff
@@ -9,17 +9,17 @@ menu:
...
 
-# Corda Network Foundation and Corda Network
+# Corda Network
 
 
 [Corda Network](https://corda.network/) is a publicly-available internet of Corda nodes operated by network participants. Each
 node is identified by a certificate issued by the network’s identity service, and is also discoverable on a network map.
 
-Corda Network went live in December 2018, and is currently governed by the independent, not-for-profit Corda Network Foundation. The Foundation's Board of Directors is made up of the network's initial participants in Spring 2019 and will oversee the Foundation until democratic elections are held. See the [governance model](https://corda.network/governance/governance-guidelines.html) for more detail.
+Corda Network went live in December 2018, and is currently governed by the independent, not-for-profit Corda Network Foundation. See the [governance model](https://corda.network/corda-network-foundation/articles-of-association/) for more detail.
```
Comment (KadriSiff) [inline content/en/platform/corda/4.5/open-source/corda-network/corda-network-foundation.md:22]: Could we use this link here: https://corda.network/corda-network-foundation/about-the-foundation

File: content/en/platform/corda/4.6/enterprise/corda-network/the-corda-network.md [hunk 2]
```diff
@@ -4,18 +4,18 @@ menu:
...
 
-# Corda Network Foundation and Corda Network
+# Corda Network
 
 
 [Corda Network](https://corda.network/) is a publicly-available internet of Corda nodes operated by network participants. Each
 node is identified by a certificate issued by the network’s identity service, and is also discoverable on a network map.
 
-Corda Network went live in December 2018, and is currently governed by the independent, not-for-profit Corda Network Foundation. The Foundation's Board of Directors is made up of the network's initial participants in Spring 2019 and will oversee the Foundation until democratic elections are held. See the [governance model](https://corda.network/governance/governance-guidelines.html) for more detail.
+Corda Network went live in December 2018, and is currently governed by the independent, not-for-profit Corda Network Foundation. See the [governance model](https://corda.network/corda-network-foundation/articles-of-association/) for more detail.
```
Comment (KadriSiff) [inline content/en/platform/corda/4.6/enterprise/corda-network/the-corda-network.md:18]: Could we use this link: https://corda.network/corda-network-foundation/about-the-foundation

File: content/en/platform/corda/4.7/enterprise/corda-network/the-corda-network.md [hunk 3]
```diff
@@ -4,18 +4,18 @@ menu:
...
 
-# Corda Network Foundation and Corda Network
+# Corda Network
 
 
 [Corda Network](https://corda.network/) is a publicly-available internet of Corda nodes operated by network participants. Each
 node is identified by a certificate issued by the network’s identity service, and is also discoverable on a network map.
 
-Corda Network went live in December 2018, and is currently governed by the independent, not-for-profit Corda Network Foundation. The Foundation's Board of Directors is made up of the network's initial participants in Spring 2019 and will oversee the Foundation until democratic elections are held. See the [governance model](https://corda.network/governance/governance-guidelines.html) for more detail.
+Corda Network went live in December 2018, and is currently governed by the independent, not-for-profit Corda Network Foundation. See the [governance model](https://corda.network/corda-network-foundation/articles-of-association/) for more detail.
```
Comment (KadriSiff) [inline content/en/platform/corda/4.7/enterprise/corda-network/the-corda-network.md:19]: https://corda.network/corda-network-foundation/about-the-foundation

File: content/en/platform/corda/4.8/open-source/corda-network/corda-network-foundation.md [hunk 4]
```diff
@@ -9,17 +9,17 @@ menu:
...
 
-# Corda Network Foundation and Corda Network
+# Corda Network
 
 
 [Corda Network](https://corda.network/) is a publicly-available internet of Corda nodes operated by network participants. Each
 node is identified by a certificate issued by the network’s identity service, and is also discoverable on a network map.
 
-Corda Network went live in December 2018. See the [Corda Network Foundation Board](https://corda.network/corda-network-foundation/foundation-board) for more detail on governance.
+Corda Network went live in December 2018, and is currently governed by the independent, not-for-profit Corda Network Foundation. See the [governance model](https://corda.network/corda-network-foundation/articles-of-association/) for more detail.
```
Comment (KadriSiff) [inline content/en/platform/corda/4.8/open-source/corda-network/corda-network-foundation.md:23]: https://corda.network/corda-network-foundation/about-the-foundation

File: content/en/platform/corda/4.6/enterprise/corda-network/the-corda-network.md [hunk 5]
```diff
@@ -41,9 +41,8 @@ signing requests (CSRs) from prospective network participants (sometimes via a b
 information submitted. A digitally signed participation certificate is returned if:
 
 
-* The participant meets the requirements specified in the [bylaws and policies](https://corda.network/policy/admission-criteria.html)
-of the foundation (broadly speaking, limited to sanctions screening only);
-* The participant agrees to Corda Network participant [terms of use](https://corda.network/participation/terms-of-use.html).
+* The participant meets the requirements specified in the [Corda Network Rulebook](https://corda.network/corda-network-rulebook/introduction);
```
Comment (ivanterziev-r3) [inline content/en/platform/corda/4.6/enterprise/corda-network/the-corda-network.md:44]:
```suggestion
* The participant meets the requirements specified in the [Corda Network Rulebook](https://corda.network/corda-network-rulebook/introduction).
```

File: content/en/platform/corda/4.3/enterprise/corda-network/UAT.md [hunk 6]
```diff
@@ -48,4 +48,4 @@ Corda Network Pre-Production is governed by an [independent Foundation](https://
 
 Steps to join are outlined on the [Corda Network microsite](https://corda.network/participation/index.html) - follow any specific instructions for ‘Pre-Production’.
```
Comment (KadriSiff) [inline content/en/platform/corda/4.3/enterprise/corda-network/UAT.md:49]: Apologies if I missed this in the earlier review - this link doesn't work and should be replaced with
https://corda.network/joining-corda-network/onboarding-workflow

File: content/en/platform/corda/4.4/open-source/corda-network/UAT.md [hunk 7]
```diff
@@ -55,4 +55,4 @@ Corda Network Pre-Production is governed by an [independent Foundation](https://
 
 Steps to join are outlined on the [Corda Network site](https://corda.network/participation/index.html).
```
Comment (KadriSiff) [inline content/en/platform/corda/4.4/open-source/corda-network/UAT.md:56]: Apologies if I missed this in the earlier review - this link doesn't work and should be replaced with
https://corda.network/joining-corda-network/onboarding-workflow

File: content/en/platform/corda/4.6/open-source/corda-network/corda-network-foundation.md [hunk 8]
```diff
@@ -44,9 +44,8 @@ signing requests (CSRs) from prospective network participants (sometimes via a b
 information submitted. A digitally signed participation certificate is returned if:
 
 
-* The participant meets the requirements specified in the [bylaws and policies](https://corda.network/policy/admission-criteria.html)
-of the foundation (broadly speaking, limited to sanctions screening only);
-* The participant agrees to Corda Network participant [terms of use](https://corda.network/participation/terms-of-use.html).
+* The participant meets the requirements specified in the [Corda Network Rulebook](https://corda.network/corda-network-rulebook/introduction);
```
Comment (ivanterziev-r3) [inline content/en/platform/corda/4.6/open-source/corda-network/corda-network-foundation.md:47]:
```suggestion
* The participant meets the requirements specified in the [Corda Network Rulebook](https://corda.network/corda-network-rulebook/introduction).
```

File: content/en/platform/corda/4.3/enterprise/corda-network/UAT.md (hunk 6 above)
Comment (AMcFarlandCampbellR3) [reply content/en/platform/corda/4.3/enterprise/corda-network/UAT.md]: Changed throughout

File: content/en/platform/corda/4.3/open-source/corda-network/UAT.md [hunk 9]
```diff
@@ -48,4 +48,4 @@ Corda Network Pre-Production is governed by an [independent Foundation](https://
 
 Steps to join are outlined on the [Corda Network microsite](https://corda.network/participation/index.html) - follow any specific instructions for ‘Pre-Production’.
```
[the remark this answers is not in the document]
Comment (AMcFarlandCampbellR3) [reply content/en/platform/corda/4.3/open-source/corda-network/UAT.md]: Changed throughout

File: content/en/platform/corda/4.4/open-source/corda-network/UAT.md (hunk 7 above)
Comment (AMcFarlandCampbellR3) [reply content/en/platform/corda/4.4/open-source/corda-network/UAT.md]: Changed throughout

File: content/en/platform/corda/4.7/enterprise/corda-network/the-corda-network.md [hunk 10]
```diff
@@ -4,18 +4,18 @@ menu:
...
```
Comment (KadriSiff) [inline content/en/platform/corda/4.7/enterprise/corda-network/the-corda-network.md]: This hasn't been updated to  https://corda.network/corda-network-foundation/about-the-foundation

File: content/en/platform/corda/4.5/enterprise/corda-network/the-corda-network.md [hunk 11]
```diff
@@ -4,18 +4,18 @@ menu:
...
 
-# Corda Network Foundation and Corda Network
+# Corda Network
 
 
 [Corda Network](https://corda.network/) is a publicly-available internet of Corda nodes operated by network participants. Each
 node is identified by a certificate issued by the network’s identity service, and is also discoverable on a network map.
 
-Corda Network went live in December 2018, and is currently governed by the independent, not-for-profit Corda Network Foundation. The Foundation's Board of Directors is made up of the network's initial participants in Spring 2019 and will oversee the Foundation until democratic elections are held. See the [governance model](https://corda.network/governance/governance-guidelines.html) for more detail.
+Corda Network went live in December 2018, and is currently governed by the independent, not-for-profit Corda Network Foundation. See the [governance model](https://corda.network/corda-network-foundation/articles-of-association/) for more detail.
```
Comment (KadriSiff) [inline content/en/platform/corda/4.5/enterprise/corda-network/the-corda-network.md]: This hasn't been updated to https://corda.network/corda-network-foundation/about-the-foundation

File: content/en/platform/corda/4.5/open-source/corda-network/corda-network-foundation.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (AMcFarlandCampbellR3) [reply content/en/platform/corda/4.5/open-source/corda-network/corda-network-foundation.md]: It has been updated. I think you somehow commented on an old version.

**Pull request merged**